Servers/Administration/OJS.Servers.Administration/wwwroot/js/mvc-grid-form.js

```diff
@@ -1,33 +1,54 @@
-$(document).ready(function() {
-    $(":checkbox").on('click', function(event) {
+$(document).ready(function () {
+    $(":checkbox").on('click', function (event) {
         if (event.target.value === "True") {
             event.target.value = "False"
         } else {
             event.target.value = "True"
         }
     });
 
+    $(".expandable").on('click', function (event) {
+        const expandElementId = $(event.target).attr("expand");
+
+        $(`#${expandElementId}`).attr('class').includes('d-none')
+            ? $(`#${expandElementId}`).removeClass('d-none')
+            : $(`#${expandElementId}`).addClass('d-none');
+    });
+
```

Comment:
Better to move this code outside of this js file, since it is part of the `mvc-grid` package. Updating this package will delete this code.